src/app/concerns/state/concerns.actions.ts

```diff
@@ -59,3 +60,11 @@ export class ResetPaginator {
 export class EnableAllocateAdmin extends EnableAllocateAdminPayload {
   static readonly type = `${label} Enable Allocate Admin`;
 }
+
```

Comment:
These following methods seems to be duplicated in connections as well. Isn't it better to extract them to a common method?

Reply:
cheers for flagging, yea just looking at this now. am consolidating the common actions in to a common place